input/data/manure/refreshed_manure_management.json

```
@@ -60,6 +60,12 @@
         "cover": "no cover",
         "storage_time_period": 365,
         "surface_area": null
+      },
+      {
+        "name": "compost_1",
+        "type": "Composting",
```

```suggestion
        "type": "composting",
```
Keys in JSONs are case-sensitive, right? If so, I would make this all lower case just to match the other types

---

So this field is used for a couple things: 
1. Telling Manure Manager which type of processor to instantiate.
2. Making the correct output prefix for outputs sent from this processor to OM.

After some discussion with Pooya and the rest of the dev team, because we want the output from the processors to look like the standardized class capitalization system (pascal case or PascalCase), we decided it would be easiest to standardize the inputs to look like what we want the outputs to eventually look like.
`Manure.Storage.Composting.niko's_composting.function.variable`.

I've got a PR with how this will look: https://github.com/RuminantFarmSystems/MASM/pull/2282

This will be enforced by IM eventually when we update the metadata properties.

---

Gotcha, I was not aware that this was a component of that PR but it makes a lot of sense. Thanks for the explanation!